jobsapp/forms.py

```diff
@@ -36,3 +40,16 @@ class ApplyJobForm(forms.ModelForm):
     class Meta:
         model = Applicant
         fields = ("job",)
+
+
+class ContactForm(forms.Form):
```

Comment:
The form and serializer (ContactSerializer) should have same definition (max_lengths, etc)